test/joystick_test.py

```diff
@@ -31,21 +37,38 @@ def todo_test_Joystick(self):
 
 
 class JoystickModuleTest(unittest.TestCase):
-    def todo_test_get_count(self):
-
-        # __doc__ (as of 2008-08-02) for pygame.joystick.get_count:
+    @unittest.skipIf(0 == JOYSTICK_COUNT, "No joysticks detected")
+    def test_get_count(self):
+        # Test get_count correctly identifies number of connected joysticks
+        prompt("Please add or remove joysticks as necessary before the test.")
 
+        pygame.joystick.init()
         # pygame.joystick.get_count(): return count
-        # number of joysticks on the system
-        #
-        # Return the number of joystick devices on the system. The count will
-        # be 0 if there are no joysticks on the system.
-        #
+        # number of joysticks on the system, 0 means no joysticks connected
+        JOYSTICK_COUNT = pygame.joystick.get_count()
+
+        self.assertGreaterEqual(JOYSTICK_COUNT, 0, ("joystick.get_count() must "
+                                            "return a value >= 0"))
+
+        response = question(
+            ("NOTE: Having Steam open may add an extra virtual joystick for "
+             "each joystick physically plugged in.\n"
+             "Is the correct number of joysticks connected to this system [{}]?"
+             .format(JOYSTICK_COUNT))
+        )
```

Comment:
Again for this message it would probably be useful to indicate that by 'joystick' pygame also refers to console controllers. Perhaps what the confirmation message is after could be a little clearer too. Something like:

> joystick.get_count() thinks there is {} joystick(s) or controller(s) connected to this system. Is this correct?